content/fr/docs/concepts/_index.md

```diff
@@ -0,0 +1,77 @@
+
+## Les objets Kubernetes 
+
+Kubernetes contient un certain nombre d'abstractions représentant l'état de votre système: applications et charges de travail conteneurisées déployées, leurs ressources réseau et disque associées, ainsi que d'autres informations sur les activités de votre cluster. Ces abstractions sont représentées par des objets dans l'API Kubernetes; voir la [vue d'ensemble des objets Kubernetes](/docs/concepts/abstractions/overview/) pour plus de détails.
+
+Les objets de base de Kubernetes incluent:
+
+* [Pod](/docs/concepts/workloads/pods/pod-overview/)
+* [Service](/docs/concepts/services-networking/service/)
+* [Volume](/docs/concepts/storage/volumes/)
+* [Namespace](/docs/concepts/overview/working-with-objects/namespaces/)
+
+En outre, Kubernetes contient un certain nombre d'abstractions de niveau supérieur appelées Contrôleurs. Les contrôleurs s'appuient sur les objets de base et fournissent des fonctionnalités supplémentaires. Ils comprennent:
+
+* [ReplicaSet](/docs/concepts/workloads/controllers/replicaset/)
+* [Deployment](/docs/concepts/workloads/controllers/deployment/)
+* [StatefulSet](/docs/concepts/workloads/controllers/statefulset/)
+* [DaemonSet](/docs/concepts/workloads/controllers/daemonset/)
+* [Job](/docs/concepts/workloads/controllers/jobs-run-to-completion/)
+
+## Plan de contrôle de Kubernetes
+
+Les différentes parties du plan de contrôle Kubernetes, telles que les processus Kubernetes Master et kubelet, déterminent la manière dont Kubernetes communique avec votre cluster. Le plan de contrôle conserve un enregistrement de tous les objets Kubernetes du système et exécute des boucles de contrôle continues pour gérer l'état de ces objets. À tout moment, les boucles de contrôle du plan de contrôle répondent aux modifications du cluster et permettent de faire en sorte que l'état actuel de tous les objets du système corresponde à l'état souhaité que vous avez poussé.
+
+Par exemple, lorsque vous utilisez l'API Kubernetes pour créer un objet Deployment, vous fournissez un nouvel état souhaité au système. Le plan de contrôle Kubernetes enregistre la création de cet objet et exécute vos instructions en lançant les applications requises tout en les planifiant sur les nœuds de cluster de manière à ce que l'état actuel du cluster corresponde à l'état souhaité.
+
+### Master Kubernetes
+
+
+Le master Kubernetes est responsable du maintien de l'état souhaité pour votre cluster. Lorsque vous interagissez avec Kubernetes, par exemple en utilisant l'interface de ligne de commande `kubectl`, vous communiquez avec le master Kubernetes de votre cluster.
+
+> Le "master" fait référence à un ensemble de processus gérant l’état du cluster. En règle générale, tous les processus sont exécutés sur un seul nœud du cluster. Ce nœud est également appelé master. Celui-ci peut également être répliqué pour la disponibilité et la redondance.
```

Comment:
redondance or résilience?

Reply:
In en doc : "The master can also be replicated for availability and redundancy."
Redundancy = redondance.